lib/diagnostic.rb

```diff
@@ -78,11 +94,12 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = "I am a Cat, and I go meow"
 
 ## Explain why this would be the output, based on the method lookup chain.
 # your response as a comment here
-##
+## because house_cat inherits the say_hello method from the animal class.
```

Comment:
Be as specific as possible.  house_cat/House_cat and animal should be `HouseCat` and `Animal`, respectively.